presets/massa-station-preset.js

```diff
@@ -81,7 +91,6 @@ module.exports = {
           fontSize: "16px",
           fontWeight: "600",
           fontFamily: theme("fontFamily.Urbane"),
-          lineHeight: "19px",
```

Comment:
This needs to be reverted to original value (19px) see [here](https://github.com/massalabs/ui-kit/pull/38) for context